controllers/routes.js

```diff
@@ -51,10 +51,6 @@ const toplevel = {
                 template: 'static-pages/region'
             })
         },
-        {
-            path: '/contact',
```

Comment:
Does this just come from the CMS now?

Reply:
This got moved to be a "top-level" section, see later on in this routes file.